grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/IdFragment.java

```diff
@@ -44,7 +45,7 @@ public void applyTraversal(GraphTraversal<Vertex, Vertex> traversal) {
 
     @Override
     public String getName() {
-        return "[id:" + idToString(id) + "]";
+        return "[id:" + idToString(id.getValue()) + "]";
```

Comment:
In fact, we might be able to make this the `ConceptId#toString` method (or some other method on `ConceptId`)